fe/fe-core/src/main/java/com/starrocks/sql/StatementPlanner.java

```diff
@@ -168,7 +169,8 @@ private static ExecPlan createQueryPlan(Relation relation,
         LogicalPlan logicalPlan;
 
         try (Timer ignored = Tracers.watchScope("Transformer")) {
-            logicalPlan = new RelationTransformer(columnRefFactory, session).transformWithSelectLimit(query);
+            TransformerContext transformerContext = new TransformerContext(columnRefFactory, session, false);
```

Comment:
what does the third argument "false" means here? some comment?

Reply:
done